docs/versioned_docs/version-4.1.x/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/rs-configuration.md

```diff
@@ -518,4 +518,4 @@ This option needs to be set to true when using a `btrfs` filesystem, if the appl
 ## See Also
 
 - [Installation](../../../quickstart-guide/installation.md)
-- [Deploy an Application](../replicated-pv-mayastor/rs-deployment.md)
```

Comment:
No clue why line 521 shows a diff - nothing changed here. I edited this online with the GH editor, perhaps it changed some whitespace?

Reply:
I am also not sure

Reply:
This is not an issue. It doesn't affect the output.